### ISSUE TITLE
Always Says "Up to Date"

### ISSUE THREAD

**Issue by ColetteDiskette** [dhruv8sh/arch-update-checker#47]
On my previous install, this worked perfectly fine. I since switched from an Nvidia card to an AMD card so I decided to just wipe and start over, and using this now always shows that I'm "up to date" even when checking in terminal via pacman or yay yields some or many updates. I'm wondering if there's a dependency I'm missing. I use yay and the checker has yay selected with the `--noconfirm` flag in the text box for flags. Use `pamac` and checking Flatpak updates are both disabled.

**Comment by dhruv8sh** [comment]
I should mention this in the README. The only dependency is pacman-contrib.

**Comment by ColetteDiskette** [comment]
Hmm, interesting. That means there's an issue elsewhere because:

```
❯ sudo pacman -S pacman-contrib
[sudo] password for colette: 
warning: pacman-contrib-1.10.6-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Packages (1) pacman-contrib-1.10.6-1

Total Installed Size:  0.12 MiB
Net Upgrade Size:      0.00 MiB

:: Proceed with installation? [Y/n] n
```

**Comment by dhruv8sh** [comment]
Okay, then go to config. Enable command debugging under debug options.
Run ```plasmashell --replace```, and print any relevant logs after you check for updates.

**Comment by ColetteDiskette** [comment]
Actually, looking at that output, I think I solved it. It's piping into `pacinfo` and I didn't have `pacutils` installed. I don't have any updates atm (I ended up accidentally updating in a misstep before posting here by clicking the "Update your system" button to see if it saw the updates in the script it ran.)

I'm pretty sure it'll work now once updates come down the line again. I appreciate your help, as well as your work on this useful Plasmoid!
